android/expoview/build.gradle

```diff
@@ -8,6 +8,10 @@ apply plugin: 'com.jakewharton.butterknife'
 group = 'host.exp.exponent'
 version = '32.0.0'
 
+def safeExtGet(prop, fallback) {
```

Comment:
Wouldn't it be possible to implement this method only once in some root gradle file? `allprojects { … }` or sth?

Reply:
Even if it would be possible to do from `allprojects` then it wouldn't be clear where this method comes from and what it does. 
It might be better If we do this in a separate file in `unimodules-core` and import it in all unimodules, but I'm not entirely sure if we really need it for just one small and simple method 🤷‍♂️